application/models/message.go

```diff
@@ -183,6 +185,90 @@ func (m *Message) Create(tx *pop.Connection, user User, requestUUID string, thre
 	return nil
 }
 
+// Todo Once gql is no longer supported, simplify the name of this function and get rid of the Create() function above
+// Create a new message if authorized.
+func (m *Message) CreateForRest(tx *pop.Connection, user User, input api.MessageInput) *api.AppError {
```

Comment:
I see your todo, so perhaps this is moot, but another option would be to name this method `CreateFromInput` which could stick around if there was need for another kind of create method.